gammapy/data/event_list.py

```diff
@@ -379,6 +380,45 @@ def filter_circular_region(self, region):
 
         return mask
 
+    def select_region(self, region, wcs):
+        """Select events in circular regions.
```

Comment:
I think `circular` is not true here, it's more general. Maybe change to `...in specified region.`?

Reply:
done